md/customize-resource-layouts.md

```diff
@@ -0,0 +1,113 @@
+
+Prerequisites to customize a layout:
+
+* Basic knowledge of Java script
+* Basic knowledge of AngularJS
+* Basic knowledge of Bootstrap
+* Basic knowledge of the UI Designer
+* An existing living application (to test the modified layout)
+
+The following example shows how to add a new widget into the layout.    
+The example sections show how to:
+* Export the layout from Bonita Portal
+* Import the layout into the UI Designer
+* Modify the layout
+* Apply the customized layout onto a living application.
+
+## Export a layout from Bonita Portal
+
+1. Open Bonita Portal, and log in with username and password of someone mapped to the Administrator profile.
+2. Click on the **Resources** menu.
+3. Select the filter **Layouts**.
+4. Select the element that you want to modify.
+5. Click on **Export** and save this layout on your file system.
+
+## Import a layout into the UI Designer
+
+To facilitate the modifications, layouts are built using the UI Designer. Therefore you just have to import it into the UI Designer to make changes.
+
+1. Open the UI Designer.
+2. Click on the **Import** button.
+3. Select the layout to import.
+
+## Edit the layout using the UI Designer
+
+When you import the layout into the UI Designer, you can see that it generates three artefacts:
+
+1. A _Layout page_, which can be found in the layout tab.
+2. The _Living application IFrame_ widget, used to display the application page, found in the custom widgets tab.
+3. An _Application menu_ widget to display the application menu, found in the custom widgets tab.
+
+So we propose here to add new widgets to the **Layout page** header and footer. 
+
+In the UI Designer:
+
+1. Click on the **Layout page** to open it in the page editor.
+2. See that this page is composed by three main containers **layout-header**, **layout-content**, **layout-footer**.
+3. Drag and drop an image widget on top of the menu in the **layout-header**, to display a brand image. 
+4. Drag and drop a title widget in the **layout-footer**, to display a "Copyright © 2017, Example Corporation"
+5. Click on **Save**.
+
+## Export the customized layout onto your file system
+
+Once your changes are made, save the new layout using a new name and then export it.
+
+1. Rename it into _myCustomLayout_.
+2. Click on **Save**.
+3. Click on the **Export** button.
+
+## Import the _myCustomLayout_ into the portal
+
+1. Open Bonita Portal, and again log in with username and password of someone mapped to the Administrator profile.
+2. Click on the **Resources** menu.
+3. Click on **Add**.
+4. Add the new _myCustomLayout_.
+5. Click **Next**.
+6. Click **Confirm**.
+
+## Configure your living application to use the modified layout
+
+1. Click on the **Applications** menu.
+2. Click on the edit action **...** of your living application.
+3. In the **Look & Feel** section, click on the **Layout** link
+4. Select the _myCustomLayout_
```

Comment:
Select _myCustomLayout_
or
Select the layout _myCustomLayout_ layout

Reply:
done